environment/env-dev

```diff
@@ -2,3 +2,4 @@ export FLASK_APP=flasksrc
 export FLASK_ENV=development
 export TOYNET_IMAGE_TAG=miniflask-dev
 export MINI_FLASK_PORT=5000
+export COMPOSE_NETWORK=reclass_network
```

Comment:
This is overwriting any custom environment variable that I am passing into the container from `docker-compose`. Could we either remove this or figure out a way to have this be overwritten? The other options aren't that big of a deal if they can't be overwritten but this one could change more often depending on the environment.

Reply:
I broke our the environment definition into the Docker image build process, rather than sourcing during container runtime. It adds a few layers of indirection to environment variables, but it should allow `docker-compose` to overwrite them.